Sources/IO/Geometry/GLTFImporter/Extensions.js

```diff
@@ -89,7 +90,10 @@ export function handleKHRLightsPunctual(extension, transformMatrix, model) {
 export async function handleKHRDracoMeshCompression(extension) {
   const reader = vtkDracoReader.newInstance();
   reader.parse(extension.bufferView);
-  return reader.getOutputData();
+  const pdn = vtkPolyDataNormals.newInstance();
```

Comment:
shouldn't you NOT compute polydatanormals if the file already contains normals ?

Reply:
Yes, i saw that the vtkDracoReader is already setting the normals but i am not sure why the result is not correct, i will see if i can find why

Reply:
If i disable the [setNormals ](https://github.com/Kitware/vtk-js/blob/89758edd084dc59dd85908afc2e49ba9d1dc4230/Sources/IO/Geometry/DracoReader/index.js#L174) in vtkDracoReader the rendering is the expected one and the normals array from GLTF and GLTF-Draco are the same size but not same values

Reply:
You might want to use the GlyphMapper to visualize the normals created by Draco. Maybe that would give a clue about what is wrong...